### ISSUE TITLE
[Unity 2021.3 MVU] Controller visualization appearing on Hololens 2 after starting a recording

### ISSUE THREAD

**Issue by Clandguth** [microsoft/MixedRealityToolkit-Unity#11347]
## Describe the bug

A single controller appeared on the users right hand after the user started recording behavior for another issue. User did not see the controller visualization before starting the recording

## To reproduce

1. Checkout the 'Enforce 2021.3 as minimum version, upgrade sample project to 2021.3.16f1' PR
2. https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11345 from Zee2:2021_upgrade
3. Build Example scenes and load them onto device
4. Launch MRT3Sample, and start recording while in any scene
5. Observe controller appears over the users right hand
6. 1/3 reproduction rate when closing and re-launching the application

## Expected behavior

No controller is shown on HL2

## Video

https://user-images.githubusercontent.com/110259320/210881563-c1d3cb89-f518-429c-9e23-ac0ccf55aa68.mp4

## Your setup (please complete the following information)

- Unity Version [2021.3.16f1]
- MRTK Version [3.13]

## Target platform (please complete the following information)

- HoloLens 2

**Comment by keveleigh** [comment]
Do you have a log available for this one?

**Comment by Clandguth** [comment]
Added to the PR directly
